packages/elements/src/icons/ui/IconsUi.stories.tsx

```diff
@@ -1,83 +1,122 @@
 import * as React from "react";
 import { IconDemo } from "./IconsUiDemo";
+import * as allIcons from "./IconsUi";
 import {
+  stenaAngleDown,
   stenaAngleLeft,
   stenaAngleRight,
   stenaAngleUp,
-  stenaAngleDown,
+  stenaArrowCircleDown,
+  stenaArrowCircleLeft,
+  stenaArrowCircleRight,
+  stenaArrowCircleUp,
+  stenaArrowDown,
   stenaArrowLeft,
   stenaArrowRight,
   stenaArrowUp,
-  stenaArrowDown,
+  stenaArrowWideDown,
   stenaArrowWideLeft,
   stenaArrowWideRight,
   stenaArrowWideUp,
-  stenaArrowWideDown,
+  stenaBell,
+  stenaBellFilled,
   stenaBusinessAgreement,
-  stenaBusinessCustomer,
   stenaBusinessCard,
   stenaBusinessClaim,
+  stenaBusinessCustomer,
   stenaBusinessInvoice,
   stenaBusinessNSLH,
-  stenaCalendarManage,
-  stenaExpand,
   stenaBusinessSignature,
-  stenaTrash,
-  stenaTimes,
-  stenaTimesCircle,
-  stenaSliders,
-  stenaSearch,
-  stenaRefresh,
-  stenaPlus,
-  stenaMinus,
-  stenaPlusCircle,
-  stenaMinusCircle,
-  stenaPen,
-  stenaHamburger,
+  stenaCalendar,
+  stenaCalendarFilled,
+  stenaCalendarManage,
+  stenaCheck,
+  stenaCheckCircle,
+  stenaCheckCircleFilled,
+  stenaClock,
+  stenaClockFilled,
   stenaCopy,
   stenaCopyDocument,
+  stenaDownload,
+  stenaExclamationCircle,
+  stenaExclamationCircleFilled,
+  stenaExclamationTriangle,
+  stenaExclamationTriangleFilled,
+  stenaExpand,
+  stenaHamburger,
+  stenaInfoCircle,
+  stenaInfoCircleFilled,
+  stenaIntermodal,
   stenaLogout,
   stenaMail,
-  stenaSwitch,
+  stenaMinus,
+  stenaMinusCircle,
   stenaMobile,
+  stenaPen,
+  stenaPhone,
+  stenaPlus,
+  stenaPlusCircle,
+  stenaRefresh,
   stenaSailingGate,
   stenaSailingLocation,
   stenaSailingOnQuay,
   stenaSailingOnRoute,
   stenaSailingShip,
   stenaSailingTrain,
-  stenaCheck,
-  stenaCalendar,
-  stenaBell,
-  stenaBellFilled,
-  stenaExclamationTriangle,
-  stenaExclamationTriangleFilled,
-  stenaExclamationCircle,
-  stenaExclamationCircleFilled,
-  stenaInfoCircle,
-  stenaInfoCircleFilled,
-  stenaCheckCircle,
-  stenaCheckCircleFilled,
-  stenaClock,
-  stenaClockFilled,
-  stenaArrowCircleDown,
-  stenaArrowCircleLeft,
-  stenaArrowCircleRight,
-  stenaArrowCircleUp,
-  stenaCalendarFilled,
+  stenaSearch,
+  stenaSliders,
+  stenaSortDown,
+  stenaSortUp,
+  stenaSwitch,
+  stenaTimes,
+  stenaTimesCircle,
+  stenaTrash,
   stenaUser,
   stenaUserCircle,
-  stenaPhone,
-  stenaSortUp,
-  stenaSortDown,
-  stenaDownload,
-  stenaIntermodal,
 } from "./IconsUi";
+import { Box, Txt } from "@stenajs-webui/core";
+import { Icon } from "../../components/ui/icon/Icon";
 
 export default {
   title: "elements/Icons/UI",
 };
 
+export const AllIconsDemo = () => (
+  <Box
+    indent
+    spacing
+    style={{
+      display: "grid",
+      gap: "8px",
+      gridTemplateColumns: "repeat(auto-fill, minmax(80px, 1fr))",
+      background: "#f0f1f3",
```

Comment:
Something wrong with ui-100? 🤔

Reply:
Copy paste from font-awesome ;) But you're right, let's use webui colors instead :)